prog/minio/minio_cluster_nexus.rb

```diff
@@ -51,14 +51,6 @@ def self.assemble(project_id, cluster_name, location, admin_user,
     end
   end
 
-  def before_run
-    when_destroy_set? do
-      unless ["destroy", "wait_pools_destroyed"].include?(strand.label)
```

Comment:
Just looking at the diff, it looks suspicious, but `wait_pools_destroyed` is only hopped to after `decr_destroy`, so this should work with the base `before_run`.  Similar for the diff in the minio_pool_nexus.

Reply:
Yes, `wait_pools_destroyed` should only be hopped from `destroy`. Since the base `before_run` checks the `destroying` semaphore instead of `strand.label`, and `destroying` is incremented at `wait_pools_destroyed`, I think it's fine.